Homework

HKV203 opened the pull request:
1. Save Tickets and Users
2. Fetch Tickets by ID and user ID and Users by ID
3. Update Tickets type
4. Delete Users by ID and their tickets (if any)

Commit:
1. Save Tickets and Users
2. Fetch Tickets by ID and user ID and Users by ID
3. Update Tickets type
4. Delete Users by ID and their tickets (if any)

VolgaDaniluk commented on src/main/java/dao/DAO.java:
```diff
@@ -0,0 +1,102 @@
+                + "' WHERE id = " + id + ";");
+    }
+
+    public void deleteUsers(int id) {
```
the logic can be simplified by using cascades (they are added during the table creation)

VolgaDaniluk commented on src/main/java/dao/DAO.java:
- use prepared statement for command with input parameters
- you have to create specific dao classes (TicketDao and UserDao)
- all strings with sql queries should be moved to specific class and be constants
-

Commit:
1. DAO class is split into several classes
2. All the queries are moved into a separate class and are finals now
3. The DELETE cascade is applied to the tickets table
